simple/kg_util/README.md

```diff
@@ -0,0 +1,22 @@
+# Libraries for working with Data Commons KG
```

Comment:
To confirm, stuff in kg_util is unedited from tools repo?

Reply:
Yes.